autotest/test_gwf_maw11.py

```diff
@@ -0,0 +1,254 @@
+import os
```

Comment:
Not critical, but I think our convention is to write a short description of the test and its purpose.